webrtc.html

```diff
@@ -5462,6 +5462,19 @@ <h3 id="sec.identity-verify-assertion">Verifying Identity Assertions</h3>
     connections between participants. One will normally assume that
     the IP address of TURN servers is not sensitive information.</p>
 
+    <p>Mitigating the exposure of IP addresses to the application requires
+    limiting the IP addresses that can be used, which will impact the
+    ability to communicate on the most direct path between endpoints.
+    Browsers are encouraged to provide appropriate controls for deciding
+    which IP addresses are made available to applications, based on the
+    security posture desired by the user.</p>
+
```

Comment:
That's why the note is on the next line saying that the WG has NOT come consensus on this.
I believe we have consensus on saying at least this, and may need to have stronger language.